stubs/app/Models/Resource.php

```diff
@@ -6,11 +6,13 @@
 use Illuminate\Database\Eloquent\Model;
 use Spatie\Sluggable\HasSlug;
 use Spatie\Sluggable\SlugOptions;
+use Spatie\Translatable\HasTranslations;
 
 class Resource extends Model
 {
     use HasFactory;
     use HasSlug;
+    use HasTranslations;
 
     /**
```

Comment:
The `language` attribute should be removed as the model's fields now support multiple languages.